backend/dataall/modules/s3_datasets/api/table/types.py

```diff
@@ -21,6 +22,15 @@
         gql.Field(name='nodes', type=gql.ArrayType(TablePermission)),
     ],
 )
+DatasetTableRestrictedInformation = gql.ObjectType(
+    name='DatasetTableRestrictedInformation',
+    fields=[
+        gql.Field(name='AwsAccountId', type=gql.String),
+        gql.Field(name='GlueDatabaseName', type=gql.String),
+        gql.Field(name='GlueTableName', type=gql.String),
```

Comment:
similar to call out above that `GlueDatabaseName` is a part of index in opensearch (`backend/dataall/modules/s3_datasets/indexers/table_indexer.py`) - not sure really how "restricted" they are treated if that information can be found elsewhere

++ to note I would think best if anything would be to restrict on open search and keep the other server side restricted logic the same

Reply:
I think this might be the solution, restricting it also in OpenSearch